.github/workflows/status_embed.yaml

```diff
@@ -0,0 +1,63 @@
+name: Status Embed
+
+on:
+  workflow_call:
+    inputs:
+      job_status:
+        required: true
+        type: string
+    secrets:
+      webhook_token:
+        required: true
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}-${{ github.sha }}
+  cancel-in-progress: true
+
+jobs:
+  status_embed:
+    # We need to send a status embed whenever the workflow
+    # sequence we're running terminates. There are a number
+    # of situations in which that happens:
+    #
+    # 1. We reach the end of the Deploy workflow, without
+    #    it being skipped.
+    #
+    # 2. A `pull_request` triggered a Lint & Test workflow,
+    #    as the sequence always terminates with one run.
+    #
+    # 3. If any workflow ends in failure or was cancelled.
+    if: >-
+      (github.workflow == 'Deploy' && ${{ inputs.job_status }} != 'skipped') ||
```

Comment:
The `Deploy` name here won't be the same on all repos. We should make this an input, or standardise the name across our repos.

Reply:
How about we go for the standardization option?